libraries/Microsoft.Bot.Builder.AI.QnA/Models/QnAMakerTraceInfo.cs

```diff
@@ -111,5 +103,9 @@ public class QnAMakerTraceInfo
         /// </value>
         [JsonProperty("rankerType")]
         public string RankerType { get; set; }
+        
+        [Obsolete]
+        [JsonProperty("metadataBoost")]
```

Comment:
>[JsonProperty("metadataBoost")] [](start = 8, length = 31)

If this is no longer used, you should mark it with [JsonIgnore] so that it is no longer serialized as well.

Reply:
Implemented.